Sources/InkCLI/main.swift

```diff
@@ -7,23 +7,60 @@
 import Foundation
 import Ink
 
-guard CommandLine.arguments.count > 1 else {
-    print("""
-    Ink: Markdown -> HTML converter
-    -------------------------------
-    Pass a Markdown string to convert as input,
-    and HTML will be returned as output. To use
-    STDIN as input, call ink with "-" as a single
-    argument, like this: '$ ink -'.
-    """)
+if CommandLine.arguments.contains(where: { $0 == "-h" || $0 == "--help" }) {
+    print(helpMessage, on: .standardOut)
     exit(0)
 }
 
-var markdown = CommandLine.arguments[1]
+if CommandLine.arguments.contains(where: { $0 == "--version" }) {
```

Comment:
```suggestion
if CommandLine.arguments.contains("--version") {
```

Reply:
This is cleaned up in #19, and I'll merge those changes into this branch once it's finalized.